src/mailadm/cmdline.py

```diff
@@ -78,10 +101,25 @@ def setup_bot(ctx, email, password, show_ffi):
     if show_ffi:
         ac.add_account_plugin(FFIEventLogger(ac))
 
+    mailadmdb = get_mailadm_db(ctx)
+    with mailadmdb.read_connection() as rconn:
+        mail_domain = rconn.config.mail_domain
+        admingrpid_old = rconn.config.admingrpid
+
     if not ac.is_configured():
-        assert email and password, (
-            "you must specify --email and --password once to configure this database/account"
-        )
+        if email and not password:
+            if email.split("@")[1] == mail_domain:
+                print("--password not specified, creating account automatically... ")
+                email, password = create_bot_account(ctx, email)
```

Comment:
isn't this rather "local e-mail address specified but password was not specified so creating a password automatically."

Reply:
Yes, but the user already knows that they specified a local e-mail address, so no need to tell them at this point.